project_tests.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+set -e
+
+proofreader src/ test/ web/
```

Comment:
I wonder if it's possible to pick out what should be included from the finder inside `.phpcs`? (There's a fix inside `app` on the Journal's that's currently being ignored by CI but happens locally.)